roles/cfg_openwrt/templates/uplink_gateway/nftables.d/21-limit-inbound.nft.j2

```diff
@@ -0,0 +1,10 @@
+{% if openwrt_version == 'snapshot' %}
+chain limit_inbound {
```

Comment:
This whole file cannot work. Feels like this is a shortcoming of fw4. All hooking nftables chains are traversed until the packet is being rejected. A allow only allows passing the next chain until the final chain is reached, only then its truly allowed. Not sure how to fix that

Reply:
someone already opened a ticket for that.
https://github.com/openwrt/openwrt/issues/9981

Reply:
Feels we cannot switch to firewall4 as of now.